vtysh.conf

```diff
@@ -0,0 +1 @@
+service integrated-vtysh-config 
```

Comment:
u/s using an empty file for this? do u run as service inside the container ?

Reply:
To test various things I run in 2 podman containers  (net=xxx) but also with --net=host.  Same image works in both cases.

```
$ sudo podman run --privileged -d --network=host --name="frrHost" -e OCPPEER="10.99.0.12" -e OCPASN="64500" -e OCPROUTERID="10.99.0.11" frr-daemonset:latest 
c96e0cc28ced03e80352e5354553156a12709b153e105ce3661bee171a82322e
$ sudo podman ps 
CONTAINER ID  IMAGE                           COMMAND  CREATED        STATUS            PORTS   NAMES
c96e0cc28ced  localhost/frr-daemonset:latest           8 seconds ago  Up 7 seconds ago          frrHost
```
I'll take it out and push again